jujugui/static/gui/src/app/subapps/browser/browser.js

```diff
@@ -740,9 +738,6 @@ YUI.add('subapp-browser', function(Y) {
       // so this method only needs these lines once switched over.
       // We need to render the sidebar view as default. This is the new design
       // in the near future we will likely just render it in the initializer.
-      if (!window.flags || !window.flags.react) {
-        this.sidebar();
```

Comment:
Can you remove this method and by extension the Sidebar view, tests (anything it uses, etc)